authors/apps/authentication/models.py

```diff
@@ -82,12 +83,6 @@ class User(AbstractBaseUser, PermissionsMixin):
     # true
     is_verified = models.BooleanField(default=False)
 
-    # A timestamp representing when this object was created.
-    created_at = models.DateTimeField(auto_now_add=True)
-
```

Comment:
why did you remove these fields

Reply:
The `created_at` and `updated_at` fields are shared by both the `User` model and `Profile` model. so i deleted them from the user model. Added them to a model file that can be shared by both apps as a user and a profile are created and updated at same times.